pkg/exchange/ftx/exchange.go

```diff
@@ -96,6 +106,18 @@ func (e *Exchange) NewStream() types.Stream {
 }
 
 func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
+	markets, err := e._QueryMarkets(ctx)
+	if err != nil {
+		return nil, err
+	}
+	marketMap := types.MarketMap{}
+	for k, v := range markets {
+		marketMap[k] = v.Market
+	}
+	return marketMap, nil
+}
+
+func (e *Exchange) _QueryMarkets(ctx context.Context) (MarketMap, error) {
```

Comment:
go uses lower case method name for private func, you can use something like `queryMarkets`